### DISCUSSION TITLE
Path to Github Action README

### DISCUSSION THREAD

**Discussion by vitorveiga** [community/community#37497] [Actions]
Hello,

Can I specify the path to a README file of a Javascript GitHub's action, in the same way, we specify the "runs.main" javascript file (action.yaml)? We are trying to create a GitHub action inside a Monorepo, which already has, in the root folder, a README.md for the all repositories packages.

**Comment by ossanna16** [comment]
Hi there @vitorveiga and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
